README.md

```diff
@@ -24,17 +24,17 @@ This module provides Cloud Trace support for Node.js applications. [Google Cloud
 
         npm install --save @google/cloud-trace
 
-2. Include and start the library at the *top of the main script of your application*. It's important that Cloud Trace is the first thing executed so that it can accurately gather data:
+2. If you are running locally, or not on the Google Cloud Platform, set the GCLOUD_PROJECT environment variable. Your project ID is visible in the [Google Cloud Console Console](https://console.cloud.google.com/project?_ga=1.258049870.576536942.1443543237), it may be something like `particular-future-12345`. If your application is [running on Google Cloud Platform](running-on-google-cloud-platform), you don't need to specify the project ID.
```

Comment:
I would put this at the end of this numbered list. Also, some rewording would help, something like...

"If your application is running outside of Google Cloud Platform, such as locally, on-premise, or on another cloud provider, you will need additional configuration. See the running elsewhere section."